spec/models/like_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  let(:user) { User.create(name: 'Laura', email: 'laura@example.com', password: 'laura123') }
```

Comment:
`[Optional]` It is better if you use [factories](https://github.com/thoughtbot/factory_bot_rails) instead of `User.create` or `User.new`.